frontend/src/job/ingest-monograph/IngestMonographMetadataForm.vue

```diff
@@ -164,6 +164,10 @@ function extractZenonId(path: string): string {
     return result[1];
 }
 
+function filterDuplicateEntry(value, index, self) {
```

Comment:
Die Funktionsparameter sollten typisiert werden. Außerdem ist `self` etwas verwirrend in dem Kontext. Das hier funktioniert z.B. und man versteht ggf. schneller worum es geht:

```ts
function filterDuplicateEntry<T>(value: T, index: number, array: T[]) {
    return array.indexOf(value) === index;
}
```